scilog/src/app/logbook/core/snippet/snippet-content/snippet-content.component.ts

```diff
@@ -144,7 +144,10 @@ export class SnippetContentComponent implements OnInit {
   }
 
   get content() {
-    return `${this._content}${this._edited}`;
+    let content = this._content ?? '';
```

Comment:
There seems to be a problem if I edit the entry multiple times as I get multiple (edited) flags. 
`test

test

test

test

(edited)
(edited)`

Reply:
ofc, and that has always been there, my bad. Fixing it